Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -251,7 +251,9 @@ try {
             if ($latestRelease) {
                 Write-Host "Using $($latestRelease.name) (tag $($latestRelease.tag_name)) as previous release"
                 $artifactsFolder = Join-Path $baseFolder "artifacts"
```

Comment:
Another option could be to create the artifacts folder in a temp folder (e.g. $ENV:RUNNER_TEMP). If we use an existing folder we don't really know what's in it. Idk if that could lead to other issues

Reply:
I actually thought about it this scenario as well. Generally, having other apps there, shouldn't matter.
And I'm not sure if there'll be side effects of using a temp folder.